### ISSUE TITLE
Change default plaintext color

### ISSUE THREAD

**Issue by Xyhlon** [jesseduffield/lazygit#1565]
**Topic**
A clear and concise description of what you want to discuss
I want to know how to change the default plain text color. 
Because I almost cannot see anything when i commit or merge stuff.
What is the proper way to change this color?

![image](https://user-images.githubusercontent.com/26318103/140506020-abcaee2f-6c8c-493d-8d55-612995f68ef1.png)

**Comment by kohane27** [comment]
![screenshot-2021-12-10-07-30-44](https://user-images.githubusercontent.com/57322459/145492572-27a3f5b9-1e95-4044-a289-a64cb21aa877.png)
I'm having the exact same issue. I think it has to do with your terminal. Are you using Alacritty? Because In my Alacritty, I have the exact same issue: I can't read the text. But in Kitty, everything looks fine. The lazygit config are the same when I opened both terminals.

My guess is that it has to do with your Alacritty theme.

**Comment by Xyhlon** [comment]
Yes I use Alacritty do you think it alacritty is the problem or lazygit
